druntime/src/object.d

```diff
@@ -104,6 +104,13 @@ else version (AArch64)
     else version = WithArgTypes;
 }
 
+// Lower Associative Array to a newaa struct for static initialization.
```

Comment:
Maybe a note here to say that this is only callable using CTFE (otherwise you will trigger the assert in the impl).

Reply:
Or, just an `assert(__ctfe);` at the beginning...